File: Python/Warmup-1/sum_double.py

# this function return the sum of the two given int values
# if the two values are the same, return double their sum
def sum_double(a, b):
	i = 0

	if a is not b:
		i = a + b
	else:
		i = (a + b) * 2

	return i

print(sum_double(1, 2))
print(sum_double(3, 2))
print(sum_double(2 ,2))

File: JavaScript/Warmup-1/sumDouble.js

// given two int values, return their sum. Unless the two values are the same, then return double their sum
const sumDouble = (a, b) => {
    return a === b ? (a + b) * 2 : a + b;
};

console.log(sumDouble(1, 2));
console.log(sumDouble(3, 2));
console.log(sumDouble(2, 2));
